carpet/metrics.py

```diff
@@ -6,32 +6,25 @@
 import warnings
 import torch
 import numpy as np
-import prox_tv
+from prox_tv import tv1_1d
+
 from .utils import init_vuz
 from .checks import check_tensor
 from .lista_analysis import ListaTV
+from .lista_analysis import LEARN_PROX_PER_LAYER
+from .loss_gradient import loss_prox_tv_analysis
 
 
 def compute_prox_tv_errors(network, x, lbda):
-    """ Returnt the sub-optimality gap of the prox-tv at each iteration. """
+    """Return the sub-optimality gap of the prox-tv at each iteration.
+    """
```

Comment:
Better?

Reply:
I usually do it that way but it doesn't really matter.